composer.json

```diff
@@ -16,6 +16,8 @@
     },
     "require": {
         "php": "^7.4",
+        "league/flysystem": "^2.0@dev",
+        "yiisoft/aliases": "^3.0@dev",
```

Comment:
Do we need aliases there?

Reply:
https://github.com/yiisoft/files/pull/12#discussion_r390593384 comments make sense. Why not getting path from alias manually _before_ passing it to Filesystem?